### ISSUE TITLE
Using CatBoost with distributed hyperparameter tuning frameworks on a compute cluster

### ISSUE THREAD

**Issue by vroomzel** [catboost/catboost#1091]
Hello!
I'd like to be able to use a distributed hyperparameter tuning framework like Amazon's SageMaker or Ray's Tune with catboost. 
There are examples of how to use XGboost and LightGBM  - both provide callbacks to monitor training progress. For example, https://github.com/ray-project/ray/blob/master/python/ray/tune/examples/lightgbm_example.py#L31

How can i implement this with catboost? Is there another way to tune catboost on a compute cluster?

Thanks,
Alex

**Comment by annaveronika** [comment]
Do you need to look on the error after training has finished? If so, you can call model.score function or call predict and after that calculate metric on predictions.

**Comment by annaveronika** [comment]
closing because of inactivity
